torch/_torch_docs.py

```diff
@@ -2311,10 +2311,14 @@ def merge_dicts(*dicts):
 .. math::
     inv = (u^T u)^{{-1}}
 
+Supports input of float, double, cfloat and cdouble dtypes. Also supports batches of matrices, and if A is a batch of matrices then the output has the same batch dimensions.
```

Comment:
nit: :math:`A`.

Reply:
Thank you for your detailed feedbacks and efforts. It is ashamed that I missed many points.